public/src/ajaxify.js

```diff
@@ -449,9 +449,11 @@ $(document).ready(function () {
 					url: ev.state.returnPath,
 				}, ev.state.returnPath, config.relative_path + '/' + ev.state.returnPath);
 			} else if (ev.state.url !== undefined) {
-				ajaxify.go(ev.state.url, function () {
-					$(window).trigger('action:popstate', { url: ev.state.url });
-				}, true);
+				require(['hooks'], function (hooks) {
```

Comment:
hooks is already defined at the top, no need to require here?

Reply:
Hooks at the top are defined in IIFE. This `document.ready` has no access to it.

Reply:
Ahh then we need another one in this IIFE, maybe right after `$(document).ready(function () {`